static/js/lib/ovs_validation.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import R from "ramda";
+import { validation, emptyOrNil, validate } from "./validation";
+
+
+export const validateCollectionForm = validate([
+  validation(emptyOrNil, R.lensPath(["title"]), "Title is required"),
```

Comment:
Are moira lists validated on the back end instead of here?

Reply:
Back end